cinema/management/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.core.management import BaseCommand, CommandError
+from django.db import connections
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        connection = None
+        while not connection:
+            try:
+                connection = connections["default"]
+            except CommandError:
+                self.stdout.write("Database unavailable")
+
+                time.sleep(1)
```

Comment:
Catching CommandError here is not going to help. Instead, you should catch the OperationalError that will be thrown when you try to use the connection and the database is unavailable.